src/Compilers/CSharp/Test/Emit2/Emit/RuntimeProbing/ModuleCancellationTests.cs

```diff
@@ -32,18 +32,18 @@ private CompilationVerifier CompileAndVerify(string source, string? expectedOutp
             expectedOutput: expectedOutput);
 
     private static void AssertNotInstrumented(CompilationVerifier verifier, string qualifiedMethodName)
-        => AssertInstrumented(verifier, qualifiedMethodName, "<PrivateImplementationDetails>.ModuleCancellationToken", expected: false);
+        => AssertNotInstrumented(verifier, qualifiedMethodName, "<PrivateImplementationDetails>.ModuleCancellationToken");
 
     private static void AssertNotInstrumentedWithTokenLoad(CompilationVerifier verifier, string qualifiedMethodName)
-        => AssertInstrumented(verifier, qualifiedMethodName, @"ldsfld     ""System.Threading.CancellationToken <PrivateImplementationDetails>.ModuleCancellationToken""", expected: false);
+        => AssertNotInstrumented(verifier, qualifiedMethodName, @"ldsfld ""System.Threading.CancellationToken <PrivateImplementationDetails>.ModuleCancellationToken""");
```

Comment:
I don't quite follow what happened here - we removed the spaces and split the IL below by space and then check if the indicator is there - how can it ever be if the IL is split by spaces and the indicator contains spaces?

Reply:
We split by spaces removing empty strings in the resulting array and then join with a single space separator. THis normalizes the number of consecutive spaces to 1.